frontend/src/views/applications/SelectFacilityView.vue

```diff
@@ -108,6 +119,13 @@ export default {
         this.$refs.form?.validate()
```

Comment:
we want to always enable the Next button. When the user clicks on the Next button, it will trigger the form validation. if there is any missing info, they will see some error messages and cannot go to the next page

Reply:
done